moped-editor/src/views/projects/projectView/ProjectMilestones.js

```diff
@@ -79,16 +80,16 @@ const ProjectMilestones = ({ projectId, loading, data, refetch }) => {
       render: (milestone) => milestone.moped_milestone.milestone_name,
       validate: (milestone) => !!milestone.milestone_id,
       editComponent: (props) => (
-        <FormControl style={{ width: "100%" }}>
+        <FormControl variant="standard" style={{ width: "100%" }}>
           <Autocomplete
             id={"milestone_name"}
             name={"milestone_name"}
             options={Object.keys(milestoneNameLookup)}
             getOptionLabel={(option) => milestoneNameLookup[option]}
-            getOptionSelected={(option, value) => option === value}
+            isOptionEqualToValue={(option, value) => option === value}
             value={props.value}
             onChange={(event, value) => props.onChange(value)}
-            renderInput={(params) => <TextField {...params} />}
+            renderInput={(params) => <TextField variant="standard" {...params} />}
```

Comment:
i'm seeing the dreaded no options match error in this component:

> MUI: The value provided to Autocomplete is invalid.
None of the options match with `9`.
You can use the `isOptionEqualToValue` prop to customize the equality test. 
 
I'm quite sure this predates this PR and is related to the fact that `Object.keys(milestoneNameLookup)` returns an array of strings when the `value` is actually a number.

fix it later?